### PR TITLE
[yang] Added Tunnel flex counter group

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### Why I did it
Fixes https://github.com/sonic-net/sonic-buildimage/issues/13457
Added Tunnel flex counter group

#### How I did it
Added relevant container in sonic-flex_counter yang model

#### How to verify it
Added UT to verify

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

File: src/sonic-yang-models/doc/Configuration.md [hunk 1]
```diff
@@ -922,6 +922,9 @@ instance is supported in SONiC.
     },
     "QUEUE": {
         "FLEX_COUNTER_STATUS": "enable"
+    },
+    "TUNNEL": {
+        "FLEX_COUNTER_STATUS": "enable"
```
Comment (qiluo-msft) [inline src/sonic-yang-models/doc/Configuration.md:927]: >FLEX_COUNTER_STATUS [](http://example.com/codeflow?start=9&length=19)

POLL_INTERVAL is missing on 4 entries. Is it expected?

Comment (dgsudharsan) [reply src/sonic-yang-models/doc/Configuration.md]: I have added examples

Comment (qiluo-msft) [reply src/sonic-yang-models/doc/Configuration.md]: Could you replace tab with spaces?

Comment (dgsudharsan) [reply src/sonic-yang-models/doc/Configuration.md]: Done

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**